sw/device/silicon_creator/lib/manifest.h

```diff
@@ -458,6 +467,71 @@ typedef struct manifest_ext_secver_write {
   uint32_t write;
 } manifest_ext_secver_write_t;
 
+/**
+ * Manifest extension: Integration Specific Firmware Binding (ISFB).
+ *
+ * The Integration Specific Firmware Binding (ISFB) extension is used to bind
+ * firmware to certain integrator-device phases (e.g. restrict firmware to
+ * unreleased integrator devices only). The `ROM_EXT` needs to perform these
+ * constraint checks instead of the firmware because the rescure protocol
+ * bypasses any firmware level enforcement.
+ *
+ * This extension also provides a mechanism to enable firmware owners to perform
+ * their own rollback and upgrade automated testing. The rollback policy is
+ * implemented on top of the `strike_bits` provided in the ISFB info flash
+ * region.
+ */
+typedef struct manifest_ext_isfb {
+  /**
+   * Required manifest header.
+   */
+  manifest_ext_header_t header;
+  /**
+   * Strikeout mask. The size of this array is equal to 128 strike bits packed
+   * into uint32_t words.
```

Comment:
We should also probably mention that a bit value of `1` indicates no constraint, so `!0` means no constraints

Reply:
Fixed. Clarified based on spec definition.